CODEOWNERS

```diff
@@ -11,3 +11,4 @@ purism/librem/13v3 @yegortimoshenko
 system76/darp6 @khumba
 framework @emiller88
 tuxedo/pulse/15/gen2 @trueNAHO
+tuxedo/pulse/14/gen3 @gabyx @britter @trueNAHO
```

Comment:
Do code owners actually work, if the owners are not committers on this repository?

Reply:
Dont know :)

Reply:
We can probably keep these lines, but github is throwing warnings for all non-nixpkgs committers.